server/router/upload.ts

```diff
@@ -18,7 +18,9 @@ router.post('/', validateUser('User'), (req, res) => {
     body: { id, tableName, fileBuffer },
   } = req;
   const validTables = ['Riders', 'Drivers', 'Admins'];
-
+  if (fileBuffer === '') {
+    res.status(400).send({ err: 'Invalid file name: empty string' });
+  }
```

Comment:
you actually need an `else` here wrapping the rest of the function, or else the rest of the code will continue to execute